tests/BrowsershotTest.php

```diff
@@ -977,6 +977,31 @@ public function it_can_send_cookies()
         ], $command);
     }
 
+    /**
```

Comment:
Make this annotation use one line, like the rest of the tests

Reply:
Done.